.github/workflows/codeql-analysis.yml

```diff
@@ -63,9 +63,8 @@ jobs:
     #    and modify them (or add more) to build your code if your project
     #    uses a compiled language
 
-    #- run: |
-    #   make bootstrap
-    #   make release
+    - run: |
+       make RELEASE=1
```

Comment:
```suggestion
       make RELEASE=1 RUNTESTS=0
```

Reply:
Thanks, I got there eventually. This is what I get for devving on two laptops and a phone.